eng/_util/cmd/run-builder/run-builder.go

```diff
@@ -167,6 +184,13 @@ func main() {
 	}
 }
 
+// enableSystemWideFIPS enables system-wide FIPS if possible and returns a state-restoring func if
+// necessary. Enabling system-wide FIPS is platform-specific and may require build tags. For
+// example, the registry APIs only exist on Windows. This var is replaced by a platform-specific
+// func in a build-tagged file. If the var is nil, run-builder has no implementation for enabling
+// system-wide FIPS.
+var enableSystemWideFIPS func() (restore func(), err error)
```

Comment:
Note from Go sync: this `var` func pointer and the `init` defined in `run-builder_windows.go` was just me being lazy and not wanting to write a `run-builder_fallback.go` file. 😄 Removing in favor of conventional "real" `func`.